### PR TITLE
Refactoring and MVVM 

### PR DESCRIPTION
**Pull request by amdazm05**
1-Please remove debug and bin folders.
2-Follow the MVVM design pattern in the commits to follow
3- Seperate UI components into different folders

### CODE REVIEW

**Comment by amdazm05** [comment]
@RaoAbdulSami needs your attention

**Comment by RaoAbdulSami** [comment]
ok Sir jee.

**Comment by RaoAbdulSami** [comment]
MVVM implemented.

File: SimplCalc/view/CalcView.xaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:SimplCalc.view"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800">
+    
+    <Grid>
+        <TextBox x:Name="result" HorizontalAlignment="Left" Height="59" Margin="107,10,0,0" TextWrapping="Wrap" Text="{Binding Path =InputTxt}" VerticalAlignment="Top" Width="370" />
+        <Button Content="4" HorizontalAlignment="Left" Height="48" Margin="109,141,0,0" VerticalAlignment="Top" Width="57" Command="{Binding getNum}" CommandParameter="4"/>
+        <Button Content="5" HorizontalAlignment="Left" Height="48" Margin="211,141,0,0" VerticalAlignment="Top" Width="57" Command="{Binding getNum}" CommandParameter="5"/>
```
Comment (amdazm05) [inline SimplCalc/view/CalcView.xaml:13]: Its good here that you used bindings

File: SimplCalc/ViewModels/MainWindowViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+using System.Threading.Tasks;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+using SimplCalc.RelayCommands;
+using SimplCalc.Model;
+using System.ComponentModel;
+
+namespace SimplCalc.ViewModels
+{
+    class MainWindowViewModel: INotifyPropertyChanged
```
Comment (amdazm05) [inline SimplCalc/ViewModels/MainWindowViewModel.cs:14]: Also keep INotifyPropertyChanged and RelayCommand class into a separate class called ViewModelBase for clarity but this is acceptable

**Review by amdazm05 — COMMENTED** [review]
Good Work ! Keep this up

**Comment by RaoAbdulSami** [comment]
Thanks

**Pull request merged**